Sources/Rendering/Core/PointPicker/test/testPointPicker.js

```diff
@@ -75,6 +80,223 @@ test.onlyIfWebGL('Test vtkPointPicker image mapper', (t) => {
   gc.releaseResources();
 });
 
+test.onlyIfWebGL('Test vtkPointPicker line source', (t) => {
+  // Create some control UI
+  const gc = testUtils.createGarbageCollector(t);
+  const container = document.querySelector('body');
+  const renderWindowContainer = gc.registerDOMElement(
+    document.createElement('div')
+  );
+  container.appendChild(renderWindowContainer);
+
+  // create what we will view
+  const renderWindow = gc.registerResource(vtkRenderWindow.newInstance());
+  const renderer = gc.registerResource(vtkRenderer.newInstance());
+  renderWindow.addRenderer(renderer);
+
+  // create what we will view
+  const lineSource = vtkLineSource.newInstance();
+  lineSource.set({
+    point0: [0, 0, 0],
+    point1: [100, 0, 0],
+    resolution: 10,
+  });
+
+  const mapper = vtkMapper.newInstance();
+  mapper.setInputConnection(lineSource.getOutputPort());
+
+  const actor = vtkActor.newInstance();
+  actor.getProperty().setPointSize(1);
+  actor.getProperty().setRepresentation(Representation.WIREFRAME);
+  actor.setMapper(mapper);
+  renderer.addActor(actor);
+
+  // now create something to view it, in this case webgl
+  const glwindow = gc.registerResource(vtkOpenGLRenderWindow.newInstance());
+  glwindow.setContainer(renderWindowContainer);
+  renderWindow.addView(glwindow);
+  glwindow.setSize(400, 400);
+
+  renderWindow.render();
+
+  // Test picker
+  const picker = vtkPointPicker.newInstance();
+  picker.setPickFromList(1);
+  picker.initializePickList();
+  picker.addPickList(actor);
+  picker.setTolerance(1.0);
+
+  const pFirst = [380 + 0.5, 200, 0];
+  picker.pick(pFirst, renderer);
+
+  const actorsFirstPoint = picker.getActors();
+  t.equal(actorsFirstPoint.length, 1);
+  t.equal(actorsFirstPoint[0], actor);
+
+  const idFirstPoint = picker.getPointId();
+  t.equal(idFirstPoint, 0);
+
+  const pLast = [20, 200 + 0.5, 0];
```

Comment:
@finetjul ok, but that is a bug right? Because I think this is the reason why the picking is not working for for one of my test models. Can this be fixed soon?

Reply:
I just checked the c++ implementation, and tolerance is added onto the bounds:
https://gitlab.kitware.com/vtk/vtk/-/blob/master/Rendering/Core/vtkPicker.cxx#L653-658

So it would make sense to do it here as well. @laurennlam do you remember why you did not do it ?

If you keep having issues, it would make sense to double check the c++/js differences. 

Hth.

Reply:
It's been a very long time. I didn't remember. 
It seems that in VTK, the code used in section you mentioned also used LOD objects, property... And this class has been implemented in the beginning of vtk.js. So it didn't exist at this moment. Maybe I bypassed it and develop a simplier way to pick.

Reply:
@goflo have you tried adding tolerance on the bounds ? does it work ?

Reply:
@finetjul no I haven't tried that and I'm not sure how to test that? Can you provide a code snippet?

Reply:
Something like this should do it:
```
     const bbox = vtkBoundingBox.newInstance();
      ...
        if (mapper) {
          bbox.setBounds(mapper.getBounds());
          bbox.inflate(tol);
        } else {
          bbox.reset();
        }

        if (
          vtkBox.intersectBox(bbox.getBounds(), p1Mapper, ray, hitPosition, t)
        ) {
```

Reply:
@goflo does #1782 help with your problem ?